plugins/rule_engines/irods_rule_language/src/configuration.cpp

```diff
@@ -474,8 +474,7 @@ int hash_rules_with_copy(const std::vector<std::string>& irods_rule_bases,
 
     hasher.digest(digest);
 
-    logger::rule_engine::debug(
-        "hash_rules_with_copy: rulebases = {}; digest = {}", fmt::join(irods_rule_bases, ","), digest);
+    log_re::debug("hash_rules_with_copy: rulebases = {}; digest = {}", fmt::join(irods_rule_bases, ","), digest);
```

Comment:
Please wrap the curly braces in square brackets (i.e. `{}` -> `[{}]`). That helps detect empty strings and has become a convention throughout iRODS code.